cub/test/catch2_test_device_transform.cu

```diff
@@ -551,19 +551,22 @@ C2H_TEST("DeviceTransform::Transform buffer start alignment",
   {
     return;
   }
+  const int offset_r = offset_a;
```

Comment:
What do we need `offset_r` for?

Reply:
It's the offset used for the `result` vector. I just use the value of `offset_a`, but I don't need to. I could pick a random value here, or something else.